libs/designsystem/modal/src/modal-wrapper/config/modal-config.ts

```diff
@@ -10,6 +10,7 @@ export interface ModalConfig {
   collapseTitle?: boolean;
   component: any;
   size?: ModalSize;
+  height?: string;
```

Comment:
```suggestion
  customHeight?: string;
```
Rename to `customHeight` as we want it to be clear that this is something that _overrides_ size